config/runtime.exs

```diff
@@ -162,6 +162,12 @@ feature_file_upload =
     String.trim(System.get_env("MW_FEATURE_IDEA_FILE_UPLOAD") || "")
   )
 
+feature_privacy_imprint_enabled =
+  Enum.member?(
+    ["true"],
+    String.trim(System.get_env("MW_FEATURE_LEGAL_PRIVACY_LINKS") || "")
```

Comment:
You don't need the whole `Enum.member?` stuff here unless I'm missing something:

```
enabled? = "true" == "MW_FEATURE_LEGAL_PRIVACY_LINKS" |> System.get_env("") |> String.trim()
```

Reply:
I'd probably also wrap the feature flagging code for consistent code. I.e. the above feature also enabled for `""` which is a bit odd